examples/add_public_raster_dataset.ipynb

```diff
@@ -249,24 +248,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "688e386b-c9ac-4598-98ea-6d8a9d52f362"
-      ]
-     },
-     "execution_count": 6,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "GeoEngineException",
+     "evalue": "BodyDeserializeError: UUID parsing failed: invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `t` at 1 at line 1 column 35",
```

Comment:
Your example notebook seems to crash. Maybe you can see why this error occurs.

Reply:
[fa335ce](https://github.com/geo-engine/geoengine-python/pull/117/commits/fa335ce1b6e64e6351e97718929409cbefd541fb)